api/db-cdr/generate-cdr/build-cloudsql-tables.sh

```diff
@@ -109,7 +109,8 @@ VALUES
 (3,'Domains',4,'Labs and Measurements','Labs and Measurements',0,0,1,3),
 (4,'Domains',5,'Observations','Observations',0,0,1,4),
 (5,'Domains',6,'Procedures','Procedures',0,0,1,5),
-(6,'Program Physical Measurements',19,'Physical Measurements','Participants have the option to provide a standard set of physical measurements as part of the enrollment process(program physical measurements)',0,0,0,8)"
+(6,'Program Physical Measurements',19,'Physical Measurements','Participants have the option to provide a standard set of physical measurements as part of the enrollment process(program physical measurements)',0,0,0,8),
+(7,'Devices',2,'Devices','Devices',0,0,1,9)"
```

Comment:
My mistake on the ticket. Suggesting change
```suggestion
(7,'Domains',2,'Devices','Devices',0,0,1,6)"
```

Reply:
I saw this, but thought "device" is different from "domain" :-), else I would have made the change you suggested...